test/Directory.Build.props

```diff
@@ -1,15 +1,25 @@
 <Project>
+  <Import Project="..\Directory.Build.props" />
+
   <PropertyGroup>
-    <_ImportParentFile>false</_ImportParentFile>
-    <_ImportParentFile Condition="$(MSBuildProjectName.StartsWith( 'Microsoft.AspNetCore.' ))">true</_ImportParentFile>
+    <DeveloperBuildTestTfms>netcoreapp2.0</DeveloperBuildTestTfms>
```

Comment:
Will change this to `netcoreapp2.1` and add `netcoreapp2.1` two lines down when I move to ASP.NET Core 2.1 dependencies.